css-two-column-layout/styles.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: karla, sans-serif;
+}
+
+h1 {
+  font-size: 50px;
+  margin-top: 50px;
+}
+
+h4 {
+  margin-top: 40px;
+  width: 59%;
+}
+
+.weight-700 {
+  font-weight: 700;
+}
+
+.height-80px {
+  height: 80px;
+}
+
+.height-112px {
+  height: 112px;
+}
+
+.container {
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.letter-space-rem {
+  letter-spacing: 0.2rem;
+}
+
+.letter-space-one {
+  letter-spacing: 1px;
+}
+
+.text-upper {
+  text-transform: uppercase;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-half {
+  flex-basis: calc(100% / 2);
+}
+
+.height-250px {
+  height: 250px;
+}
+
+.flex {
+  display: flex;
+}
+
+img {
+  height: 120px;
+}
+
+/* No longer needed
+
+.img-container {
+  width: 120px;
+}
+
+*/
```

Comment:
If code is not needed, it should be deleted, not just commented.

Reply:
Okay I removed the commented out code. Not sure if I need to submit a screenshot of the output again because it's still the same.